plugins/woocommerce/client/admin/client/order-attribution-install-banner/order-attribution-install-banner.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * External dependencies
+ */
+import { Button, Card, CardBody } from '@wordpress/components';
+import { useEffect, useCallback } from '@wordpress/element';
+import { plugins, external } from '@wordpress/icons';
+import { __, sprintf } from '@wordpress/i18n';
+import { Text } from '@woocommerce/experimental';
+import { recordEvent } from '@woocommerce/tracks';
+import { getPath } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import OrderAttributionInstallBannerImage from './order-attribution-install-banner-image';
+import { useOrderAttributionInstallBanner } from './use-order-attribution-install-banner';
+import './style.scss';
+
+const WC_ANALYTICS_PRODUCT_URL =
+	'https://woocommerce.com/products/woocommerce-analytics';
+
+export const OrderAttributionInstallBanner = ( {
+	bannerImage = <OrderAttributionInstallBannerImage />,
+	eventContext = 'analytics-overview',
+	isHeaderBanner = false,
+	isSmallBanner = false,
+	showBadge = true,
+	dismissable = true,
+	title = 'Discover what drives your sales',
+	description = 'Understand what truly drives revenue with our powerful order attribution extension. Use it to track your sales journey, identify your most effective marketing channels, and optimize your sales strategy.',
+	buttonText = 'Try it now',
```

Comment:
OPinionated. Tho I believe better to keep this without defaults. So we can just send the translated text in the big component and avoid doing 

```jsx
<<OrderAttributionInstallBanner
				bannerImage={ null }
				title={ null }
			/>
```